docs/components/cursor.md

```diff
@@ -80,6 +80,7 @@ AFRAME.registerComponent('cursor-listener', {
 | downEvents  | Array of additional events on the entity to *listen* to for triggering `mousedown` (e.g., `triggerdown` for vive-controls).  | []                               |
 | fuse        | Whether cursor is fuse-based.                                                                                              | false on desktop, true on mobile |
 | fuseTimeout | How long to wait (in milliseconds) before triggering a fuse-based click event.                                             | 1500                             |
+| mode     | Camera (gaze) or mouse controlled. | gaze
```

Comment:
The problem is that gaze and mouse are mutually exclusive because the raycaster configuration is different (hence the mode property). A cursor as it is, cannot operate in both gaze and mouse modes. A `mouseMode` flag is confusing because it conveys an additional functionality on top of the normal behavior that can be toggled. We could make the `raycaster` a multiple component and the cursor component could add two of them: One for gaze and the other for mouse. This might introduce confusion since configuring the raycaster by the user would not be as simple as:
```javascript
el.setAttribute('raycaster', 'objects', '.clickable')
```
Two different raycasters would have to be configured instead:
```javascript
 el.setAttribute('raycaster'__gaze, 'objects', '.clickable');
 el.setAttribute('raycaster'__mouse, 'objects', '.clickable');
```
An alternative would be to add an `objects` property to the `cursor` component so the user does not configure the raycaster anymore